core-bundle/src/Resources/contao/drivers/DC_Table.php

```diff
@@ -3554,7 +3559,7 @@ protected function treeView()
 			$label = $GLOBALS['TL_LANG']['MOD']['page'][0];
 		}
 
-		$icon = $GLOBALS['TL_DCA'][$table]['list']['sorting']['icon'] ?: 'pagemounts.svg';
+		$icon = $GLOBALS['TL_DCA'][$table]['list']['sorting']['icon'] ?? 'pagemounts.svg';
```

Comment:
Same here.

Reply:
Fixed in 1550fd769ce350112719147253a37736f5c7cd79.